examples/btc-to-evm-fungible-transfer/.env.sample

```diff
@@ -0,0 +1,13 @@
+SYGMA_ENV=testnet
+BLOCKSTREAM_URL="your blockstream url"
+DESTINATION_ADDRESS="your evm destination address"
+DESTINATION_DOMAIN_CHAIN_ID="destination domain chain id"
```

Comment:
We can just call it `DESTINATION_CHAIN_ID`, domain is providing unnecessary context here IMO